src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -649,7 +649,7 @@ static EpInfo buildEpInfoFromStats(
     EpInfo epInfo = new EpInfo(rowCount, new HashMap<>());
```

Comment:
pl cleanup the setDefaultValues from the whole callstack and from the parameter provider.

Reply:
ignore this, the stacked PR I'm sending actually uses this param.